janusgraph-dist/src/assembly/static/cassandra/conf/jvm-server.options

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 JanusGraph Authors
```

Comment:
The file has been renamed, but the `README.md` in `janusgraph-dist` still uses the old name.